geoext.ux/ux/SimplePrint/lib/GeoExt.ux/SimplePrint.js

```diff
@@ -429,10 +430,12 @@ GeoExt.ux.SimplePrint = Ext.extend(Ext.form.FormPanel, {
      *  Do the print.
      */
     print: function() {
+        // TODO cancel ongoing print job if any
         if (this.printProvider.supportProgress()) {
             var self = this;
             var interval = null;
             var updateStatus;
+            this.progressPanel.removeAll();
```

Comment:
I think I now understand what you originally wanted to do. You want that the user can submit multiple print requests and that there is a list with all the ongoing and finished requests. This wasn't communicated very well in the UI. For me the UI simply looked broken.

Is this required by the spec that the user can submit multiple print requests at the same time, or should a previous print request be cancelled if the user hits "Print" again? @ybolognini ?